docs/generators/apex.md

```diff
@@ -290,7 +290,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 | ---- | --------- | ---------- |
 |BasicAuth|✓|OAS2,OAS3
 |ApiKey|✓|OAS2,OAS3
-|OpenIDConnect|✗|OAS3
+|OpenIDConnect|✓|OAS3
```

Comment:
do these generators really support OpenIDConnect as part of the PR?

Reply:
Nope, but the pr check list said : **Commit all changed files.**.

I wanted to separate preparatory work from generators work, cut into small chuncks ;)